.gitignore

```diff
@@ -1,2 +1,2 @@
-bower_components
 node_modules
+.vscode
```

Comment:
Don't we have a preference for people to use their local global ignore file for their own environment-related stuff, and limit the project ignore files for stuff that is generated by the project itself?

Reply:
TBH, I didn't know that was a thing until reading this comment!

https://stackoverflow.com/questions/7335420/global-git-ignore#7335487